Description: Fix wrong radio buttons when changing shortcuts
 When the dialog box was displayed, the radio buttons were not
 initialized correctly if a client-type action was to be modified.
Author: ska67 <git17@ska67.de>
Origin: backport
Bug: https://github.com/lxde/lxqt/issues/1376
Applied-Upstream: commit:a9ec3c0
Last-Update: 2018-02-05
--- a/config/edit_action_dialog.cpp
+++ b/config/edit_action_dialog.cpp
@@ -197,6 +197,14 @@ bool EditActionDialog::load(qulonglong i
         else
         {
             command_PTE->clear();
+
+            // To disable all radio boxes, once one has been selected, we need to turn exclusivity off temporarily
+            command_RB->setAutoExclusive(false);
+            dbus_method_RB->setAutoExclusive(false);
+            command_RB->setChecked(false);
+            dbus_method_RB->setChecked(false);
+            command_RB->setAutoExclusive(true);
+            dbus_method_RB->setAutoExclusive(true);
         }
 
         description_LE->setEnabled(canEdit);
